packages/flutter_tools/templates/module/ios/library/Flutter.tmpl/projectName.podspec.tmpl

```diff
@@ -0,0 +1,13 @@
+Pod::Spec.new do |s|
+  s.name                  = '{{projectName}}'
+  s.version               = '0.0.1'
+  s.summary               = 'Flutter module'
+  s.description           = 'Flutter module - {{projectName}}'
+  s.homepage              = 'https://flutter.dev'
+  s.license               = { :type => 'BSD' }
+  s.author                = { 'Flutter Dev Team' => 'flutter-dev@googlegroups.com' }
+  s.source                = { :path => '.' }
+  s.ios.deployment_target = '8.0'
+  s.vendored_frameworks   = 'App.framework'
+  s.dependency 'Flutter'
```

Comment:
Here we're assuming that the engine is the one coming from the cocoapods master spec repo? Or should this point to a specific local path?

Reply:
podspecs don't let you specify a path.  It's up to the Podfile to do that.  Which is part of the reason we should take control of the Flutter CocoaPod.

Reply:
How does this work exactly? So the outer project's Podfile went to a specific directory and found the the App.framework's pod because your install_flutter_application_pod function told it where to look. Then how does the App.framework find specifically where the matching version of the Flutter.framework comes from? 

It seems like it's coming from this line but wouldn't this just go to the online pod spec github repo to find the published engine?

Reply:
As far as I can tell, the Podfile is the driver, and the dependencies between the podspecs are resolved after all the explicit pods are installed.  The podspec doesn't try to do anything, it just describes how the targets should be made dependent on each other in the build phases.  So once all the pods are installed for a target, if there are missing pods, they will be queried and fetched from the CocoaPods' repo.
So because there's a `pod 'Flutter', :path => ` in the Podfile, nothing additional is downloaded because the App podspec dependency is satisfied.
![Screen Shot 2019-07-25 at 1 53 40 PM](https://user-images.githubusercontent.com/682784/61908058-a9bc2600-aee3-11e9-816a-499e174d7d01.png)

Reply:
Ah doh, I missed that line. Thanks for explaining. 

Can we add a comment here saying this acts as a requirement rather than a fulfillment mechanism and that it shouldn't do anything if podhelper does what it should do.

Reply:
I think that's just what a podspec is, CoocaPods has pretty good docs for all this.